src/static/css/2019.css

```diff
@@ -495,6 +495,13 @@ header.alt-bg a:focus, footer.alt-bg a:focus {
   align-items: center;
 }
 
+.year-2020 .btn,
+.year-2020 nav ul .toc,
+.year-2020 nav ul .contributors,
+.year-2020 nav ul .methodology {
+  display:none;
```

Comment:
We should have a 2020 base template that simply omits these links, rather than hiding with CSS.

Reply:
Until we decide how different 2019 and 2020 will be I say no. These are temporarily hidden - I fully expect to unhide them when launching 2020. So I don't think it's worth cloning the 2019 to 2020 templates just to remove these. If there are other reasons to create the 2020 templates (like it is a different design) then we can revisit this but for not I'd rather not have the extra maintenance when they will be pretty much direct clones.

Reply:
I'd be OK to have the following in the `base/2019/base.html` template if you think this is better than CSS hiding - would also avoid us having to add the class:

```
        <ul>
          {% if year != '2020' %}
          <li><a href="{{ url_for('table_of_contents', year=year, lang=lang) }}" class="toc">{{ self.table_of_contents_title() }}</a></li>
          <li><a href="{{ url_for('contributors', year=year, lang=lang) }}" class="contributors">{{ self.contributors_title() }}</a></li>
          <li><a href="{{ url_for('methodology', year=year, lang=lang) }}" class="methodology">{{ self.methodology_title() }}</a></li>
          {% endif %}
          <li>
            {{ year_switcher('footer') }}
          </li>
          <li>
            {{ language_switcher('footer') }}
          </li>
        </ul>
```

Reply:
OK figured out a better way to do this in https://github.com/HTTPArchive/almanac.httparchive.org/pull/893/commits/d5b2e678a96e63644b8697f213b4489e29805576